test/jest-puppeteer/testSequencer.js

```diff
@@ -0,0 +1,30 @@
+const Sequencer = require('@jest/test-sequencer').default;
+
+class CustomSequencer extends Sequencer {
+  // /**
+  //  * Select tests for shard requested via --shard=shardIndex/shardCount
```

Comment:
From my understanding sharding allows for a test suite to be split into arbitrary chunks. It's useful for lengthy test suites, not the case here. Then tests are identified via shardIndex/shardCount. 
I left the sharding commented since it was part of the TestSequencer code in the Jest docs. Since the tests aren't being split using shard flag when I ran the tests with or without the sharding section, everything worked as intended.

FYI - Info that I found on sharding:
- https://medium.com/@mfreundlich1/speed-up-your-jest-tests-with-shards-776e9f02f637
- https://jestjs.io/docs/configuration#testsequencer-string